mcodingbot/plugins/python.py

```diff
@@ -10,6 +10,26 @@
 PEP_REGEX = re.compile(r"pep *(?P<pep>\d{1,4})", re.IGNORECASE)
 
 
+DISMISS_BUTTON_ID = "dismiss"
+
+
+def encode_dismiss_button_id(id: hikari.Snowflake):
+    return f"{DISMISS_BUTTON_ID}:{id}"
+
+
+def decode_dismiss_button_id(dismiss_button_id: str) -> hikari.Snowflake:
+    return hikari.Snowflake(dismiss_button_id.split(":")[1])
+
+
+def get_dismiss_button(id: hikari.Snowflake) -> hikari.impl.CommandBuilder:
```

Comment:
CommandBuilder? not hikari.ActionRow?

Reply:
think it needs to be `hikari.api.ActionRowBuilder` after checking with mypy

Reply:
hikari.ActionRow doesn't exist

Reply:
hikari.api.ActionRowBuilder sounds right